### DISCUSSION TITLE
Table recolor

### DISCUSSION THREAD

**Discussion by diiv0** [SiimKirjanen/seatreg#72] [Q&A]
Hello guys, i want to have an option where it could be possible to change the color of table based on how many people have booked it. Like for example. the table has place for 10, and if only 6 books in the table, then theres still space for 4 other people.. So i want when the table is full, it becomes red, as it already does now, but if its half full, it would be for example orange or similar. 

Sincerly 

Divo

**Comment by diiv0** [comment]
And sorry for my english

**Reply by SiimKirjanen** [reply]
Hello. This plugin does not have table feature. Only seats. And each seat can be booked by one person.
